.ci/setup.sh

```diff
@@ -40,15 +40,21 @@ if ! sudo docker version | grep -q "$docker_version" && [ "$CI" == true ]; then
 	"${cidir}/../cmd/container-manager/manage_ctr_mgr.sh" docker install -f
 fi
 
-if [ "$arch" = x86_64 ]; then
+case "$arch" in
+	x86_64)
 	if grep -q "N" /sys/module/kvm_intel/parameters/nested; then
 		echo "enable Nested Virtualization"
 		sudo modprobe -r kvm_intel
 		sudo modprobe kvm_intel nested=1
 	fi
-else
+	;;
+	aarch64)
+	info "CI running in bare machine"
```

Comment:
Q. Doesn't `arm` support nested ?

Reply:
I think @kalyxin02 said not, or at least not on the slave machine we have for CI.

Reply:
great! thanks for the info @Weichen81